android/app/src/main/res/values/styles.xml

```diff
@@ -12,6 +12,7 @@
         <item name="popupTheme">@style/AppTheme.Popup</item>
         <item name="android:spinnerDropDownItemStyle">@style/TextViewSpinnerDropDownItem</item>
         <item name="android:alertDialogTheme">@style/AlertDialogTheme</item>
+        <item name="android:navigationBarColor">@color/black</item>
```

Comment:
Just curious, at what point do we want black and why?

Reply:
This is just by default to override all automatic OS color manipulations 🙃 

But this color is used on login page - do you want to choose a different one?

![telegram-cloud-photo-size-2-5280970699604879249-y](https://github.com/user-attachments/assets/3a3b338f-5c93-4526-b7d7-fecb4c305882)

The reason why I decided that black is good enough is because below we have an illustration so monochromatic green color may look a little bit inconsistent.

Reply:
Ah I got it. What do we use for the rest of the app when we are in dark mode?

Reply:
@shawnborton `productDark100` which is `#061B09`. And in light mode we use `produceLight100` which is `#FCFBF9`

Reply:
Just to put more confidence: we use `theme.appBG`, and from the code:

```ts
const darkTheme = {
    // Figma keys
    appBG: colors.productDark100,
```

We use the same color as main background 👀

Reply:
Cool, I would love to see what it looks like, I think staying consistent with product colors makes more sense here.

Reply:
Nice, I think productDark100 looks great personally - cc @Expensify/design for a gut check here, but I say let's do that instead of black.

Reply:
I'm into it too!

Reply:
Same. Dig it

Reply:
Cool, looks like everyone agree to use `productDark100` - I pushed these changes, so this PR is ready for review then 😊